jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -226,7 +226,7 @@ def _update_link(self, msg={}):
             raise ValueError("Link type can only be changed after existing subsets "
                              f"are deleted, but {len(self.app.data_collection.subset_groups)} "
                              f"subset(s) still exist. To delete them, you can use "
-                             f"`delete_subsets()` from the plugin API.")
+                             f"`app.delete_subsets()`")

```

Comment:
I think this change should be reverted. Moving the stuff from orientation to app is for internal usage. Should not affect subset plugin.

Reply:
Oh wait, this is the orientation plugin. Hmm. Did we talk about moving the delete subsets button somewhere else too? @kecnry , what was your vision in https://jira.stsci.edu/browse/JDAT-4917 and how much of that still applies?

Reply:
i'll leave it for now since we still have delete_subsets in the orientation plugin (see my top level comment about possibly deprecating this)

Reply:
yeah, eventually individual plugins may have their own methods for this, when needed, but I think for now we can point to subset_tools (or wherever it **is** exposed publicly).

Reply:
theres no delete_subsets in subset_tools, just in app.py. do you want it in that plugin as well (in this ticket or a follow up)? we would have to make some decisions about naming things.  delete_subsets uses 'subset_labels' but get_regions uses '‘list_of_subset_labels'. there are probably other names for inputting subset names that we would want to unify eventually too.

Reply:
ok, but this leaves orientation's `delete_subsets` in place and public, right?  It just internally redirects that to `app.delete_subsets`?  If so, I think that is fine for now and is still an internal code cleanup, and we can do further work later if/when implementing the mixin.